notool flag does not work with bootstrap command

It's looks like same cause with #769

```
goagen bootstrap -d github.com/goa-cellar/design --notool
exit status 2
flag provided but not defined: -notool
Usage of app:
  -design string

  -force

  -notest

  -out string

  -pkg string
    	 (default "app")
  -regen

  -tooldir string
    	 (default "tool")
  -version string

panic: flag provided but not defined: -notool

goroutine 1 [running]:
flag.(*FlagSet).Parse(0xc4201095c0, 0xc42008e100, 0x4, 0x4, 0x0, 0x0)
	/Users/naoya/.gobrew/versions/1.9/src/flag/flag.go:926 +0x116
github.com/goadesign/goa/goagen/gen_app.Generate(0x0, 0x0, 0x1698740, 0x16f36e0, 0xc420076058)
	/Users/naoya/.go/src/github.com/goadesign/goa/goagen/gen_app/generator.go:53 +0x3a4
main.main()
	/Users/naoya/.go/src/github.com/goa-cellar/goagen340317514/main.go:27 +0x64
```

Thanks.

--- a/goagen/gen_app/generator.go
+++ b/goagen/gen_app/generator.go
@@ -38,7 +38,7 @@ type Generator struct {
 func Generate() (files []string, err error) {
 	var (
 		outDir, toolDir, target, ver string
-		notest, regen                bool
+		notest, notool, regen        bool
 	)
 
 	set := flag.NewFlagSet("app", flag.PanicOnError)
@@ -48,6 +48,7 @@ func Generate() (files []string, err error) {
 	set.StringVar(&ver, "version", "", "")
 	set.StringVar(&toolDir, "tooldir", "tool", "")
 	set.BoolVar(&notest, "notest", false, "")
+	set.BoolVar(&notool, "notool", false, "")
 	set.BoolVar(&regen, "regen", false, "")
 	set.Bool("force", false, "")
 	set.Parse(os.Args[1:])

--- a/goagen/gen_main/generator.go
+++ b/goagen/gen_main/generator.go
@@ -46,7 +46,7 @@ type Generator struct {
 func Generate() (files []string, err error) {
 	var (
 		outDir, toolDir, designPkg, target, ver string
-		force, regen                            bool
+		force, notool, regen                    bool
 	)
 
 	set := flag.NewFlagSet("main", flag.PanicOnError)
@@ -55,6 +55,7 @@ func Generate() (files []string, err error) {
 	set.StringVar(&target, "pkg", "app", "")
 	set.StringVar(&ver, "version", "", "")
 	set.StringVar(&toolDir, "tooldir", "tool", "")
+	set.BoolVar(&notool, "notool", false, "")
 	set.BoolVar(&force, "force", false, "")
 	set.BoolVar(&regen, "regen", false, "")
 	set.Bool("notest", false, "")

--- a/goagen/gen_swagger/generator.go
+++ b/goagen/gen_swagger/generator.go
@@ -37,7 +37,7 @@ type Generator struct {
 func Generate() (files []string, err error) {
 	var (
 		outDir, toolDir, target, ver string
-		regen                        bool
+		notool, regen                bool
 	)
 
 	set := flag.NewFlagSet("swagger", flag.PanicOnError)
@@ -45,6 +45,7 @@ func Generate() (files []string, err error) {
 	set.StringVar(&ver, "version", "", "")
 	set.String("design", "", "")
 	set.StringVar(&toolDir, "tooldir", "tool", "")
+	set.BoolVar(&notool, "notool", false, "")
 	set.StringVar(&target, "pkg", "app", "")
 	set.BoolVar(&regen, "regen", false, "")
 	set.Bool("force", false, "")

